tests/unit/Term/AliasGroupListTest.php

```diff
@@ -170,20 +171,13 @@ public function testGivenInvalidLanguageCode_removeByLanguageThrowsException( $l
 	}
 
 	public function testGivenEmptyGroups_constructorRemovesThem() {
-		$enGroup = new AliasGroup( 'en', array( 'foo' ) );
-
 		$list = new AliasGroupList( array(
 			new AliasGroup( 'de' ),
-			$enGroup,
-			new AliasGroup( 'en' ),
-			new AliasGroup( 'nl' ),
-		) );
-
-		$expectedList = new AliasGroupList( array(
+			new AliasGroup( 'en', array( 'foo' ) ),
 			new AliasGroup( 'en' ),
 		) );
 
-		$this->assertEquals( $expectedList, $list );
+		$this->assertEquals( new AliasGroupList(), $list );
```

Comment:
This was meant as an obvious cleanup for an obvious mistake, and to make fragile tests more robust. Favor `assertSame` when possible, because it is more robust. Favor assertions that compare the results of two **different** calls, because this is more robust. Now this patch does have 15 comments, but they all fail to tell me what I should do to get this unstuck. Resubmit it? Split it? But how? Revert more? But what? I find this highly frustrating.